### PR TITLE
docs: Add READMEs

### PR DESCRIPTION
**Pull request by briandipalma**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 # caplin-dev-tools
-## Packages for developing Caplin applications using node.js tooling
 
-## Setup
+## Packages for developing Caplin applications using node.js tooling
 
 This monorepo contains tens of packages, they are designed to be used as part of
```
Comment (ianss) [inline README.md]: This monorepo contains tens of packages that are designed to be used as part of

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 
 ## caplin-cli
 
-For information on the `caplin-cli` tool [read its README.md](https://github.com/caplin/caplin-dev-tools/blob/master/cli/README.md).
+For information on the `caplin-cli` tool
+[read its README.md](https://github.com/caplin/caplin-dev-tools/blob/master/cli/README.md).
 
 ## Releasing packages
 
 We are using https://github.com/conventional-changelog/conventional-changelog
```
Comment (ianss) [inline README.md]: Surround the URL in '<' and '>' so that the Markdown processor automatically makes it a link.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 
 ### Creation of CHANGELOG.md
 
-If the package you are modifying has no `CHANGELOG.md` to create a 
+If the package you are modifying has no `CHANGELOG.md` to create a
```
Comment (ianss) [inline README.md]: If the package you are modifying has no `CHANGLOG.md` file, then create a

File: README.md [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
 `CHANGELOG.md` first `cd` into the package folder and run:
```
Comment (ianss) [inline README.md]: `CHANGELOG.md` file, `cd` into the package folder, and run the command:

**Comment by briandipalma** [comment]
Updated based on comments, the commits should be squashed during merge. I think all comments have been addressed.

**Comment by briandipalma** [comment]
Is this good to merge?

**Pull request merged**